.dockerignore

```diff
@@ -1 +1,2 @@
 .github
+monitoring
```

Comment:
If we ignore this dir, will docker know when the grafana dashboard json changes then?

Reply:
Docker doesn't need to know, dashboard is deployed separately
(mounted as docker-compose volume locally and deployed as k8s object in the cluster)